pyproject.toml

```diff
@@ -22,6 +22,7 @@ line_length = 120
 [tool.pydocstyle]
 convention = "google"
 match = "(?!python/test).*"
+add-ignore = "D105"
```

Comment:
@JamesFCoffey This disables the requirement that "magic methods" like `__repr__` need to be documented. I think documenting them is silly since they have well defined semantics at the language level. How do you feel about this?